### PR TITLE
add listtransactions function for address #188

### PR DESCRIPTION
**Pull request by xinyu-li-cb**
- [PSDK-441] Arbitrary Message Signing Support on WalletAddress (#203)
- camel case
- [PSDK-441] createPayloadSignature for Wallet Class (#207)
- add listtransactions function for address

### What changed? Why?
add listtransactions function for address

```
> import { Address } from "./src/coinbase/address";
undefined
> const address = new Address("base-sepolia", "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789")
undefined
> let txns = await address.listTransactions({limit: 5})
undefined
> txns.transactions[0].blockHeight()
'14637830'
```

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: src/tests/e2e.ts [hunk 1]
```diff
@@ -112,7 +112,7 @@ describe("Coinbase SDK E2E Test", () => {
     console.log("Fetching address historical balances...");
     const balance_result = await unhydratedWallet.getDefaultAddress()?.listHistoricalBalances( {assetId: Coinbase.assets.Eth, limit: 2} );
     expect(balance_result?.historicalBalances.length).toBeGreaterThan(0);
-    console.log(`Fetched historical balances: ${balance_result?.historicalBalances.toString()}`);
```
Comment (amethystens) [inline src/tests/e2e.ts]: Balances[0].amount.toString

File: src/tests/transaction_test.ts [hunk 2]
```diff
@@ -208,6 +228,27 @@ describe("Transaction", () => {
...
+      expect(transaction.blockHeight()).toEqual(blockHeight);
+    });
+  });
+
+  describe("#content", () => {
+    it("returns the ethereum transaction", () => {
+      const transaction = new Transaction(onchainModel);
+      expect(transaction.content()).toEqual(ethereumContent);
+    });
+  });
```
Comment (John-peterson-coinbase) [inline src/tests/transaction_test.ts:250]: can we please add tests for when these values are undefined.

File: src/coinbase/transaction.ts [hunk 3]
```diff
@@ -88,6 +88,32 @@ export class Transaction {
...
+   * Returns the Block Hash where the Transaction is recorded.
+   *
+   * @returns The Block Hash
+   */
+  blockHash(): string | undefined {
+    return this.model.block_hash;
+  }
+
+  /**
+   * Returns the transaction content of the transaction (only have ethereum transaction for now).
```
Comment (John-peterson-coinbase) [inline src/coinbase/transaction.ts:110]:
```suggestion
   * Returns the Content of the Transaction.
```

We can enumerate language support in the developer documentation

File: src/tests/address_test.ts [hunk 4]
```diff
@@ -36,6 +36,109 @@ describe("Address", () => {
     });
   });
 
+  describe("#listTransactions()", () => {
```
Comment (John-peterson-coinbase) [inline src/tests/address_test.ts:39]:
```suggestion
  describe("#listTransactions", () => {
```

**Review by John-peterson-coinbase — APPROVED** [review]

**Pull request merged**